CMakeLists.txt

```diff
@@ -53,6 +53,22 @@ else()
     set( iodaconv_pygrib_ENABLED False)
 endif()
 
+# Optional:  eccodes for Python (and C libs)
+option(USE_ECCODES "Use Python eccodes package" OFF)
+if ( USE_ECCODES )
+    find_package( eccodes QUIET )
+    if ( eccodes_FOUND )
+        message(STATUS "Adding python tests that import eccodes")
+        set( iodaconv_eccodes_ENABLED True)
+    else()
+        message(STATUS "Not-found: eccoodes - eccodes disabled")
+        set( iodaconv_eccodes_ENABLED False)
+    endif()
+else()
+    message(STATUS "Skipping python tests that import eccodes")
+    set( iodaconv_eccodes_ENABLED False)
+endif()
+
```

Comment:
This section looks great! Thanks for adding it in. I tried it on my Mac and it works just fine.